File: 프로그래머스/lv1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.py
def solution(lottos, win_nums):
    answer = []
    z_cnt = 0
    a_cnt = 7
    
    for l in lottos:
        if l == 0:
            z_cnt += 1
        else:
            for w in win_nums:
                if l==w: 
                    a_cnt -= 1
    
    if z_cnt == 0:
        answer.append(6 if(a_cnt>6) else a_cnt) 
        answer.append(6 if(a_cnt>6) else a_cnt) 
    else:
        answer.append(a_cnt - z_cnt) 
        answer.append(6 if(a_cnt>6) else a_cnt) 
        
    return answer

File: 프로그래머스/lv1/77484. 로또의 최고 순위와 최저 순위/로또의 최고 순위와 최저 순위.java
class Solution {
    public int[] solution(int[] lottos, int[] win_nums) {
        int[] answer = new int[2];
        
        int z_cnt = 0;
        int a_cnt = 7;
        
        for (int l: lottos){
            if (l == 0){
                z_cnt += 1;
            }else{
                for (int w: win_nums){
                    if (l == w){
                        a_cnt -= 1;
                    }
                }
            }
            
        }
        
        if(z_cnt == 0){
            answer[0] = (a_cnt > 6) ? 6 : a_cnt ;
            answer[1] = (a_cnt > 6) ? 6 : a_cnt ;
        }else{
            answer[0] = a_cnt - z_cnt;
            answer[1] = (a_cnt > 6) ? 6 : a_cnt ;
        }
        
        
        return answer;
    }
}